arch/Cargo.toml

```diff
@@ -14,3 +14,4 @@ memory_model = { path = "../memory_model" }
 
 [dev-dependencies]
 rand = ">=0.5.5"
+cpuid = { path = "../cpuid" }
```

Comment:
I find it a bit strange for the arch crate to depend on `cpuid`. I will take a look at the code to see if maybe we can find another way of testing setup_sregs.

Reply:
I could remove the `if` based on the cpu vendor and just do 
```
sregs.gs.g = 1;
```
all the time in `test_setup_sregs`

What do you think ?

Reply:
Sounds good for the moment. We can re-visit the decision in the future, but right now I don't see any disadvantage in doing just that.

Reply:
Ok. Done.